src/Emulator.cpp

```diff
@@ -80,13 +71,14 @@ void Emulator::Stop()
 void Emulator::Render()
 {
     // Clear window
-    SDL_SetRenderDrawColor(m_renderer, 0, 0, 0, 255);
-    SDL_RenderClear(m_renderer);
+    SDL_SetRenderDrawColor(m_renderer.get(), 0, 0, 0, 255);
```

Comment:
Yeah, now to get the "pointer" that the unique_ptr points to, you have to use .get().